### PR TITLE
[Doc] Fix SimpleList properties table is too small

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/List.md [hunk 1]
```diff
@@ -2213,7 +2213,7 @@ const CustomList = () => {
 
 For mobile devices, a `<Datagrid>` is often unusable - there is simply not enough space to display several columns. The convention in that case is to use a simple list, with only one column per row. The `<SimpleList>` component serves that purpose, leveraging [material-ui's `<List>` and `<ListItem>` components](https://material-ui.com/components/lists/). `<SimpleList>` is an **iterator** component: it gets an array of ids and a data store from the `ListContext`, and iterates over the ids to display each record.
 
-### Properties
+#### Properties
```
Comment (fzaninotto) [inline docs/List.md:2216]: I don't understand that change. What's the problem?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/List.md (hunk 1 above)
Comment (WiXSL) [reply docs/List.md:2216]: `SimpleList` html props table has smaller font-size than the other props tables.
This change should fix that

simplelist-fields

Comment (fzaninotto) [reply docs/List.md:2216]: But then you're changing the document structure... The proper fix would be to change the CSS, which apparently uses a different size for tables following a h4 than a h3.

Comment (WiXSL) [reply docs/List.md:2216]: Ok, I'll look into it

**Pull request closed without merging**